### ISSUE TITLE
[Usable - Samples-> Inputs & Validation]: Increase/Decrease spin buttons under Quantity Spin button are not accessible in Scan mode.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7415]
https://microsoft.visualstudio.com/OS/_workitems/edit/30961026
